lookout/style/format/tests/test_quality_report.py

```diff
@@ -144,7 +146,7 @@ def test_eval_aggregate(self):
         output = "\n".join(output)
         output = output[:output.find("# Model report for https://github.com/jquery/jquery")]
         metrics = _get_precision_recall_f1_support(output)
-        self.assertEqual(
+        self.assertAlmostEqual(
```

Comment:
The precision allows the check to be exact. If we float in distant signs, there is an error in our code because the process must be fully deterministic.

Reply:
It was `0.8397747184701835` vs `0.8397747184701836`

Reply:
And not for all python versions.